integration-cli/docker_cli_start_test.go

```diff
@@ -187,3 +187,54 @@ func TestStartPausedContainer(t *testing.T) {
 
 	logDone("start - error should show if trying to start paused container")
 }
+
+func TestStartMultipleContainers(t *testing.T) {
+	defer deleteAllContainers()
+	// run a container named 'parent' and create two container link to `parent`
+	cmd := exec.Command(dockerBinary, "run", "-d", "--name", "parent", "busybox", "top")
+	if out, _, err := runCommandWithOutput(cmd); err != nil {
+		t.Fatal(out, err)
+	}
+	for _, container := range []string{"child_first", "child_second"} {
+		cmd = exec.Command(dockerBinary, "create", "--name", container, "--link", "parent:parent", "busybox", "top")
+		if out, _, err := runCommandWithOutput(cmd); err != nil {
+			t.Fatal(out, err)
+		}
+	}
+
+	// stop 'parent' container
+	cmd = exec.Command(dockerBinary, "stop", "parent")
+	if out, _, err := runCommandWithOutput(cmd); err != nil {
+		t.Fatal(out, err)
+	}
+	cmd = exec.Command(dockerBinary, "inspect", "-f", "{{.State.Running}}", "parent")
+	out, _, err := runCommandWithOutput(cmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+	if out != "false\n" {
```

Comment:
I wonder if we should be careful with matching on "`\n`" directly? Notice other tests use `Trim` and include `\n` and `\r` and even quotes if it makes sense.  Given Windows is running integration tests, it very well could be CRLF.